### ISSUE TITLE
kendo-scheduler-toolbar does not have display:flex applied. 

### ISSUE THREAD

**Issue by smchargue** [telerik/kendo-themes#1825]
Doing a custom build of bootstrap 4.5.2 and kendo themes using kendo-thtme-boostrap.  only customizing colors. 

In Comparing a CSS build using @progress/kendo-theme-bootstrap@4.12.0 to 4.17.0 and 4.19.0  i noticed that the k-scheduler-toolbar does not have display;flex applied like it does in 4.12.0 

When i rebuild with 4.12 the kendo scheduler function as expected 

![Capture](https://user-images.githubusercontent.com/4064621/92254132-05eacb00-ee96-11ea-937b-15122e46467c.JPG)

in kendo-theme-default/scss/scheduler/_layout.scss i found this code:
``` css
kendo-scheduler .k-scheduler-toolbar {
        @include use-background-clip();
        padding: $toolbar-padding-y $toolbar-padding-x;
        border-style: solid;
        display: flex;
        flex-direction: row;
        flex-wrap: wrap;
        align-items: center;
        justify-content: space-between;
    }
```
Doesn't the code above need .kendo-scheduler .k-scheduler-toolbar?  or is this a sass syntax i don't recognize?

**Comment by smchargue** [comment]
sorry, how do i delete this?

**Comment by joneff** [comment]
I am assuming you are using kendo-jquery, correct? If that's the case, recently we updated the styles for the upcoming 2020 R3, which is literally days away.

With that release, we introduced a couple of major changes:
* remove all custom styles form scheduler toolbar and rely on regular toolbar styles (#1520)
* introduce a new `stable` channel for themes that is more or less tied to kendo-jquery releases.

I should note that all non-dev versions are stable. Perhaps we could have use `jquery` channel for the themes and we may do so in the forseable future. But the point is that there is this channel and we'll update it a few days prior a release, so developers can simply do `npm install @progress/kendo-theme-*@stable` and have the version for the latest kendo-jquery release so styles and rendering can be compatible.

All that being said, you are just using a newer version of the themes and the rendering provided by kendo-jquery you have is lagging behind. Hence the issues. If you update when 2020 R3 is out, you will not have said issues.
